src/main/java/Main.java

```diff
@@ -0,0 +1,14 @@
+//import java.io.File;
+import java.io.File;
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        DirectoryLister lister = new DirectoryLister();
+        String folder = lister.giveAddress();
```

Comment:
It should work but we have to create one DirectoryLister per directory. I don't like the fact how you overall built this class because it lets me get new folder from user but never resets the sum. If you want to have one DirectoryLister per directory you should take in folder path as constructor param. Also getting path to the folder is not really a part of DirectoryLister and should be done in Main class.

Reply:
Remember about SOLID. DirectoruyLister should be responsible for only one thing. Which is reading the file from given path and count all lines. Getting path from the user is completely different thing.